samples/drivers/display/boards/mimxrt595_evk_cm33.conf

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
We should always try to avoid board specific overlays because it adds maintenance overhead. We have cleaned up display controller samples a few times. Please revise if it can be done on the board level.

Reply:
I've moved the majority of this overlay into the board level- the only configuration I think should remain in this sample is the heap size, as this sample specifically allocates a large buffer from the heap for the display data. Other samples (like the LVGL demo) do not do this, and don't require additional heap size